lib/mpl_toolkits/axes_grid1/axes_divider.py

```diff
@@ -291,18 +291,18 @@ def add_auto_adjustable_area(self, use_axes, pad=0.1, adjust_dirs=None):
 class AxesLocator:
     """
     A callable object which returns the position and size of a given
-    AxesDivider cell.
+    `.AxesDivider` cell.
     """
 
     def __init__(self, axes_divider, nx, ny, nx1=None, ny1=None):
         """
         Parameters
         ----------
-        axes_divider : AxesDivider
+        axes_divider : `~mpl_toolkits.axes_grid1.axes_divider.AxesDivider`
```

Comment:
Why did this become the full path with a ~ instead of `.AxesDivider` like above?

Reply:
I read the guidelines....

"Use full references `~matplotlib.colors.Normalize` with an abbreviation tilde in parameter types. "

Reply:
Ah, my mistake; sorry for the noise.